src/store/index.js

```diff
@@ -11,14 +11,24 @@ import {
   loadState,
   saveState
 } from '../helpers/localStorage';
+import { shuffle } from '../helpers/shuffleArray';
 
-export const answers = patterns.map(pattern => ({
-  ...pattern,
-  answered: false,
-  correct: null,
-  answerId: null,
-  uuid: uuid()
-}));
+const patternsWithUuids = patterns.map(pattern => ({ ...pattern, uuid: uuid() }));
+const answers = patternsWithUuids.map(current => {
+  // get 3 random patterns in addition to correct one
+  const allOtherAnswers = patternsWithUuids.filter(pattern => pattern.uuid !== current.uuid);
+  const additional = shuffle(allOtherAnswers).slice(0, 3);
+  // shuffle the 4 answers
+  const variants = shuffle([current, ...additional]);
```

Comment:
`answers` should be tied to `variants` in order to show result incorrect answers